README.md

```diff
@@ -1,7 +1,7 @@
 My Lovely Project
 =================
 
-A great example repo you can fork to your heart’s content, that serves as a launchpad for working your way through the amazing **[Learning GitHub](http://shop.oreilly.com/category/videos/programming.do)** video training series.
+A **great** example repo you can fork to your heart’s content, that serves as a launchpad for working your way through the amazing **[Learning GitHub](http://shop.oreilly.com/category/videos/programming.do)** video training series.
```

Comment:
Inserting comments at the blue plus sign (+) is VERY useful, but not easy to find.  Hover over the text and click on the plus sign.